content/en/docs/languages/java/intro.md

```diff
@@ -0,0 +1,175 @@
+| [opentelemetry-java-examples](https://github.com/open-telemetry/opentelemetry-java-examples)               | End-to-end code examples demonstrating a variety of patterns using the API, SDK, and instrumentation | n/a                                | n/a                                  | n/a                                                                                                                                                       |
+
+`opentelemetry-java`, `opentelemetry-java-instrumentation`, and
+`opentelemetry-java-contrib` each publish large catalogs of artifacts. Please
+consult repositories for details, or see the "Managed Dependencies" column in
+the [Bill of Materials](#dependencies-and-boms) table to see a full list of
+managed dependencies.
+
+As a general rule, artifacts published the same repository have the same
+version. The exception to this is `opentelemetry-java-contrib`, which can be
+thought of as a group of independent projects that are co-located in the same
+project to take advantage of shared tooling. For now, the artifacts of
+`opentelemetry-java-contrib` are aligned but this is a coincidence and will
+change in the future.
+
+The repositories have a release cadence which mirrors their high level
+dependency structure:
+
+- `opentelemetry-java` is the core and releases first each month.
+- `opentelemetry-java-instrumentation` depends on `opentelemetry-java` and is
+  next to publish.
+- `opentelemetry-java-contrib` depends on `opentelemetry-java-instrumentation`
+  and `opentelemetry-java` and is last to publish.
+- Although `semantic-conventions-java` is a dependency of
+  `opentelemetry-java-instrumentation`, it is an independent artifact with an
+  independent release schedule.
+
+## Dependencies and BOMs
+
+A
+[bill of materials](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#bill-of-materials-bom-poms),
+or BOM for short, is an artifact that helps keep the versions of related
+dependencies aligned. OpenTelemetry Java publishes several BOMs catering to
+different use cases, listed below in order of increasing scope. We highly
+recommend using a BOM. Because the BOMs are hierarchical, adding dependencies on
+multiple BOMs is not recommended, as it is redundant and can lead unintuitive
+dependency version resolution.
+
+Click the link in the "Managed Dependencies" column to see a list of the
+artifacts managed by the BOM.
+
+| Description                                                                                  | Repository                           | Group ID                           | Artifact ID                               | Current Version                            | Managed Dependencies                                      |
```

Comment:
The "Managed Dependencies" column of this table is awesome. It links to the current version of each BOM's `pom.xml`, allowing the user to see the list of managed dependencies at a glance, guaranteed to be up to date.